config/prow/cluster/hook_deployment.yaml

```diff
@@ -48,6 +48,7 @@ spec:
         - --github-token-path=/etc/github/oauth
         - --config-path=/etc/config/config.yaml
         - --job-config-path=/etc/job-config
+        - --endpoint=/hook
```

Comment:
I don't think we want to make this configurable and this is a backwards-incompatible change

Reply:
I was thinking about that. Maybe it's better to default the flag to "/hook" and change it in the external plugins in the follow-up refactors